### PR TITLE
Add links to OneFormer and InstructPix2Pix notebooks

### PR DESCRIPTION
**Pull request by l-bat**
### Changes

Added links to `OneFormer` and `InstructPix2Pix` OpenVINO notebooks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -284,6 +284,8 @@ A collection of ready-to-run Jupyter* notebooks are available to demonstrate how
 - [NNCF Post-Training Optimization of BLIP Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/233-blip-visual-language-processing)
 - [NNCF Post-Training Optimization of DeepFloyd IF Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/238-deepfloyd-if)
 - [NNCF Post-Training Optimization of Grammatical Error Correction Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/214-grammar-correction)
+- [NNCF Post-Training Optimization of OneFormer Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/249-oneformer-segmentation)
```
Comment (MaximProshin) [inline README.md:287]: Also add MMS please https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/255-mms-massively-multilingual-speech
@nikita-savelyevv , any other notebooks updated by you recently?

Comment (MaximProshin) [reply README.md:287]: + Dolly

Comment (nikita-savelyevv) [reply README.md:287]: Yes, MMS and Dolly are missing

Comment (l-bat) [reply README.md:289]: Added

**Comment by l-bat** [comment]
run pylint pre-commit tests

**Review by MaximProshin — APPROVED** [review]

**Pull request merged**